packages/commercetools/api-client/src/helpers/createAccessToken/index.ts

```diff
@@ -38,24 +39,24 @@ const refreshTokenFlow = async (config: ApiConfig): Promise<Token> => {
 
 const isTokenExpired = (token: Token): boolean => Date.now() > token.expires_at;
 
-const createAccessTokenFlow = async (config: ApiConfig, options: FlowOptions = {}): Promise<Token> => {
+const createAccessToken = async (options: FlowOptions = {}): Promise<Token> => {
```

Comment:
It can return `Promise<undefined>` as well

Reply:
but it's a valid typing as well, I don't want to put there Token | null | undefined, because it basically won't change anything